components/desktop/mate/mate-icon-theme-faenza/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Till Wegmueller
+# Copyright 2016 Ken Mays
+#
+
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		mate-icon-theme-faenza
+COMPONENT_VERSION=	1.14.0
+COMPONENT_PROJECT_URL=	http://www.mate-desktop.org
+COMPONENT_SUMMARY=	Collection of Faenza and Faience icons for the MATE desktop
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:d912ffc86e01e71e96499c993cc429922ffd2576abc075e5c212cd95095fb984
+COMPONENT_ARCHIVE_URL=	http://pub.mate-desktop.org/releases/1.14/$(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.xz
+COMPONENT_LICENSE= GPLv2, LGPLv2, FDLv1.1
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
+COMPONENT_CLASSIFICATION = System/Libraries
+COMPONENT_FMRI = library/desktop/mate/$(COMPONENT_NAME)
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+PATH=/usr/gnu/bin:/usr/bin
+
+COMPONENT_PREP_ACTION= cd $(@D)  && NOCONFIGURE=1 ./autogen.sh
+
+COMPONENT_PRE_CONFIGURE_ACTION = ( cp -raf $(SOURCE_DIR)/* $(@D))
+
+CONFIGURE_OPTIONS+= --sysconfdir=/etc
+CONFIGURE_OPTIONS+= --libexecdir=/usr/lib/mate
+CONFIGURE_OPTIONS+= --localstatedir=/var/lib
+
+CONFIGURE_ENV+= PYTHON="$(PYTHON)"
+
+COMPONENT_BUILD_ENV += CC="$(CC)"
+COMPONENT_BUILD_ENV += CFLAGS="$(CFLAGS)"
+
+# common targets
+build:		$(BUILD_64)
+
+install:	$(INSTALL_64)
+
+test:		$(NO_TESTS)
```

Comment:
please, fill REQUIRED_PACKAGES (note, gmake REQUIRED_PACKAGES detects runtime deps, but we need build time deps)